zp-relayer/validateTx.ts

```diff
@@ -207,6 +207,34 @@ async function checkRoot(proofIndex: BN, proofRoot: string, state: PoolState) {
   return null
 }
 
+async function checkScreener(address: string) {
+  if (config.screenerUrl === null || config.screenerToken === null) {
+    return null
+  }
+
+  const ACC_VALIDATION_FAILED = 'Internal account validation failed'
+
+  try {
+    const rawResponse = await fetch(config.screenerUrl, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+        'Authorization': `Bearer ${config.screenerToken}`,
```

Comment:
@LyzeOfKiel @r0wdy1 do you think it is useful to forward the support id to the screener as well?

Reply:
This probably would require changes in screener to save it somewhere, as well. Btw, @r0wdy1 is screener connected to sentry?